experimental/BLTUnfold/produceUnfoldingHistograms.py

```diff
@@ -177,14 +185,8 @@ def main():
                 # Variable names in tree
                 genSelection = '( unfolding.GenSelection == 1 )'
                 genWeight = '( unfolding.puWeight )'
-                # PU weights dodgy for 7 TeV, but should be v. close to 1
-                if options.is7TeVData:
-                    genWeight = '( 1 )'
                 offlineSelection = '( unfolding.OfflineSelection == 1 )'
                 offlineWeight = '( unfolding.bTagWeight * unfolding.puWeight )'
-                # PU weights dodgy for 7 TeV, but should be v. close to 1
-                if options.is7TeVData:
-                    offlineWeight = '( unfolding.bTagWeight )'
```

Comment:
Are pu weights ok for 7 TeV?  If so they should be included in offlineWeight (but small change).